README.md

```diff
@@ -18,7 +23,7 @@ data "template_file" "container_instance_cloud_config" {
 }
 
 module "container_service_cluster" {
-  source = "github.com/azavea/terraform-aws-ecs-cluster?ref=2.0.0"
+  source = "github.com/azavea/terraform-aws-ecs-cluster?ref=3.0.0"
```

Comment:
I am anticipating this would be part of a `3.0.0` release. 

In retrospect, this might be better to include within the `3.0.0` commit of the `release/3.0.0` branch.